test/e2e/feature_compatibility_version_upgrade/feature_compatibility_version_upgrade_test.go

```diff
@@ -1,71 +0,0 @@
-package feature_compatibility_version_upgrade
```

Comment:
How come we're getting rid of this test?

Reply:
I squashed them with `feature_compatibility_version`. They were exactly the same copy of them apart from a single additional test. Now they all live in `feature_compatibility_version`.